server/src/games/Games.ClankCatacombs/ClankCatacombsScoringModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace BGS.Games.ClankCatacombs;
+
+internal record ClankCatacombsScoringModel
+{
+    public bool isEscapedFromDepths { get; init; }
+
+    public byte ArtefactsScore { get; init; }
+    
+    public byte AllTokensScore { get; init; }
```

Comment:
Minor: I think the word "All" in this context is confusing. I suggest we remove it
```suggestion
    public byte TokensScore { get; init; }
```